src/datasource/specs/utils.spec.ts

```diff
@@ -90,21 +88,20 @@ describe('Utils', () => {
   });
 
   describe('splitTemplateQuery()', () => {
-
     // Backward compatibility
     it('should properly split query in old format', (done) => {
       let test_cases = [
         {
           query: `/alu/./tw-(nyc|que|brx|dwt|brk)-sta_(\w|\d)*-alu-[0-9{2}/`,
-          expected: ['/alu/', '/tw-(nyc|que|brx|dwt|brk)-sta_(\w|\d)*-alu-[0-9{2}/']
+          expected: ['/alu/', '/tw-(nyc|que|brx|dwt|brk)-sta_(w|d)*-alu-[0-9{2}/'],
```

Comment:
`\w\d` got turned into `wd`. i think this is a correct prettier-transform, but the original text looks very regex-ish... maybe the "before" situation was already wrong?

Reply:
you are right the query to escape w properly needs double backslash.